### ISSUE TITLE
Users Opinion feedback creation and json api exposure

### ISSUE THREAD

**Issue by imsickofmaps** [praekeltfoundation/go-nike-ghr#26]
This is a clone of M&L Quizzes (issue #9) as its a poll of users opinion with the exception that the minimum number of questions is 1. 

There should only be one API endpoint:

**GET /api/opinions/view/**

Returns a dict of the active opinions to vote on.

```
{
"opinions": {
    "opinion_view_1": {
        "start": "o_1",
        "views": {
            "o_1": {
                "opinion": "I think something really clever",
                "choices": [
                    [
                        "o_2",
                        "Yes, I agree"
                    ],
                    [
                        "o_2",
                        "No way"
                    ]
               ]
           },
           "o_2": {
                "opinion": "I think something really stupid",
                "choices": [
                    [
                        "main_menu",
                        "Yes, I agree"
                    ],
                    [
                        "main_menu",
                        "No way"
                    ]
               ]
           },
        }
   }
}
```

**Update**
- max(opinions + Choices) <= 160
- returns all active opinions
- minimum number of choices = 1

**Comment by gitumarkk** [comment]
@imsickofmaps I realized I made a mistake with the issue as I was about to publish in that I included I published it in issue 15 instead of creating a new issue hope its not a problem, if it is should I revert back changes and create a new feature?

**Comment by imsickofmaps** [comment]
Landed with #15
